sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -33,6 +33,11 @@
     <PackageReference Include="System.Memory.Data" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
```

Comment:
Why do I suddenly need this?

Reply:
Maybe they were transitive dependencies from some of the other packages before but aren't any longer.